### PR TITLE
Update lxqt-config-session_it.desktop

### PR DESCRIPTION
**Pull request by stefonarch**

### CODE REVIEW

File: lxqt-config-session/translations/lxqt-config-session_it.desktop [hunk 1]
```diff
@@ -1,4 +1,3 @@
 # Translations
-Comment[it]=Configura la sessione LxQt
-GenericName[it]=Impostazioni della sessione LxQt
-Name[it]=Impostazioni della sessione LxQt
+Name[it]=Impostazioni della sessione
+Comment[it_IT]=Configura il modulo sessione di LXQt
```
Comment (palinek) [inline lxqt-config-session/translations/lxqt-config-session_it.desktop]: This is probably a typo... there should by [it] (in \*_it file), not?

Comment (luis-pereira) [inline lxqt-config-session/translations/lxqt-config-session_it.desktop]: Yeap

**Comment by paulolieuthier** [comment]
@stefonarch please squash the commits.

**Comment by stefonarch** [comment]
I would like to do so but sorry but I am not   expert enough in using
git and afraid to make only a mess.

Il 21/05/2015 13:32, Paulo Lieuthier ha scritto:

**Comment by paulolieuthier** [comment]
@stefonarch I see. Don't worry. The easiest way, IMO, is the following:

```shell
git reset --soft HEAD~2
git commit -m "Update lxqt-config-session_it.desktop"
git push origin patch-9 -f
```
`HEAD~2` means two commits behind. You'll make a new commit and force-push into your branch. That's not the most correct or advanced way to do it, but it'll work. Don't worry if it gets a mess, eventually you'll learn it.

**Comment by stefonarch** [comment]
I never logged in with git from console, always using web interface, so
you say I just log in and paste these commands?

Il 21/05/2015 14:13, Paulo Lieuthier ha scritto:

**Comment by paulolieuthier** [comment]
@stefonarch go to the folder (using the command line) where your fork is. make sure it's clean using `git status`, and make sure you are in the correct branch using `git branch`. Then run the commands I posted. The last command will ask for your login and password.

**Comment by stefonarch** [comment]
That's the point, I was working mostly on the files in my AUR git
packages folder and copy&pasted it in the browser. I'm definitevely too
unfamiliar with using git in the correct  way, too many things to learn
for me, sorry.

Any idea when pootle will be ready?

Il 21/05/2015 16:45, Paulo Lieuthier ha scritto:

**Comment by paulolieuthier** [comment]
That's a good question. @jleclanche do have an idea?

**Comment by paulolieuthier** [comment]
Merged in 899323032a1f9a399f7015b3d1db865fece11159.

**Pull request closed without merging**